src/VisualStudio/VisualStudioInteractiveComponents/Roslyn.VisualStudio.InteractiveComponents.csproj

```diff
@@ -66,7 +66,8 @@
     <NuGetPackageToIncludeInVsix Include="System.Runtime.CompilerServices.Unsafe" />
   </ItemGroup>
 
-  <Import Project="$(VSToolsPath)\vssdk\Microsoft.VsSDK.targets"/>
+  <Import Project="$(VSToolsPath)\vssdk\Microsoft.VsSDK.targets" 
+          Condition="Exists('$(VSToolsPath)\vssdk\Microsoft.VsSDK.targets') and '$(DesignTimeBuild)' != 'true' and '$(BuildingForLiveUnitTesting)' != 'true'"/>
```

Comment:
😕 Why not just the Exists check?

Reply:
We don't need to do any of the work that is implemented in the targets during design time build.

Reply:
I would expect the VSSDK targets themselves to avoid doing unnecessary work during DTB. Is this not the case?

Reply:
Unfortunately, they do not. Microsoft.VsSDK.targets add following unconditionally:

```xml
    <PrepareForRunDependsOn>
      $(PrepareForRunDependsOn);
      GeneratePkgDef;
      CopyPkgDef;
      CreateVsixContainer;
      DeployVsixExtensionFiles;
      CopyVsixManifestFile;
      CopyVsixExtensionFiles;
    </PrepareForRunDependsOn>
```

Reply:
DTB does not do a PrepareForRun.

Reply:
Please tell me DTB does not do PrepareForRun. That would be bad.

Reply:
IDK, doesn't it?

Reply:
VSSDK is doing something during design time build: https://dev.azure.com/devdiv/DevDiv/_workitems/edit/365685?fullScreen=false

Reply:
You are right. DTB doesn't run PrepareForRun. I guess there are other ways VSSDK is hooking into the build.